src/raylib.h

```diff
@@ -86,6 +86,9 @@
 // NOTE: Microsoft specifiers to tell compiler that symbols are imported/exported from a .dll
 #if defined(_WIN32)
     #if defined(BUILD_LIBTYPE_SHARED)
+        #if defined(__TINYC__)
+            #include <_mingw.h>
```

Comment:
Sorry, in `raylib.h` I try to avoid any non-standard include.

Reply:
Should we inline it like this?
```c
#if defined(__TINYC__)
    #define RLAPI __attribute__(dllexport)
#else
    #define RLAPI __declspec(dllexport)
#endif
```

Reply:
yes, way better

Reply:
Done :smile:

Reply:
Wait! There is some errors

Reply:
`RLAPI` is redefined in `rlgl.h`

Thus I changed the declaration in `raylib.h` to be

```c
#if defined(__TINYC__)
    #define __declspec(x) __attribute__((x))
#endif
#define RLAPI __declspec(dllexport)
```

Tested with `make clean && make RAYLIB_LIBTYPE=SHARED` and running some examples and everything works just fine now.

Thank you.